x-pack/plugins/apm/public/components/app/Settings/agent_configurations/List/index.tsx

```diff
@@ -245,7 +245,9 @@ export function AgentConfigurationList({
         items={configurations}
         initialSortField="service.name"
         initialSortDirection="asc"
-        initialPageSize={20}
+        initialPageSize={25}
+        pageSizeOptions={[10, 25, 50]}
```

Comment:
I think most of these components use a component we own (e.g. ManagedTable). Does it make sense to remove props from these components as much as possible and set more sensible defaults in ManageedTable et al?